### ISSUE TITLE
Cant Move Tabs in Navbar

### ISSUE THREAD

**Issue by ghost** [piroor/treestyletab#3160]
I just freshly installed the Addon int he latest FF Version and i cant drag and drob tabs in the regular navbar anymore. While i like the functionality of the tree style i still like to shuffle around tabs so i have certain pages or topics closer together. Drag and Drop also seems unavailable in the Tree Style Tabs. This is fine as long as the regular moving in the navbar works but just after installment this functionality broke. Sending Tabs to the Top or Bottom is also not enough options.

tldr; I like to move my tabs atleast via the default FF functionality if not within the Addon itself but cant do anymore

Even after uninstalling the addon it seems like i still cant move my tabs anymore which is worrysome

**Comment by ghost** [comment]
I restarted Firefox a few times and did all kind of oother things and just as randomly as it started it stopped. No idea why. Works even with the addon.
